### ISSUE TITLE
Why I can't use this in other components except app component

### ISSUE THREAD

**Issue by TakaXhuliano** [NetanelBasal/ngx-content-loader#10]
I tried to do integrate this in other components and it doesn't work. I don't get any errors and I am checking from my chrome console that it is not rendered in the same way like in the app component that works fine, by the way.

**Comment by NetanelBasal** [comment]
My only guess is that you are missing the import in other modules.

**Comment by TakaXhuliano** [comment]
Nope, because It gets rendered in  the DOM but in some way that I don't understand why, and it would give in error if it was not imported.

Thank you

**Comment by TakaXhuliano** [comment]
I just fix it, thank you

**Comment by kvandake** [comment]
@TakaXhuliano, what are the steps were made to use in other components?
